pkg/service/service.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/edgexfoundry/device-sdk-go/v2/internal/clients"
 	"github.com/edgexfoundry/device-sdk-go/v2/internal/common"
 	"github.com/edgexfoundry/device-sdk-go/v2/internal/container"
-	"github.com/edgexfoundry/device-sdk-go/v2/internal/controller"
+	restController "github.com/edgexfoundry/device-sdk-go/v2/internal/controller/http"
```

Comment:
why is alias needed here? I don't see any name conflicts. Alisa should only be used to avoid name conflicts.

Reply:
this file imports `net/http` package, used by `AddRoutes` method. 
any idea about a better alias name ?

Reply:
Ohh, I missed that the name conflict is on `http`, not `controller`. I am good with this.